src/librustc/ty/print/pretty.rs

```diff
@@ -932,90 +959,153 @@ pub trait PrettyPrinter<'tcx>:
                     p!(write("{}{}", data, ui_str))
```

Comment:
I think the suffix here (and for signed integers) shouldn't be printed if `print_ty` is false.